test/functionalTest/cases/0001_federation/federation.test

```diff
@@ -0,0 +1,218 @@
+
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+
+--NAME--
+Federation
+
+--SHELL-INIT--
+dbInit CB_A
+dbInit CB_B
+dbInit CB_C
+brokerStart CB_A
+brokerStart CB_B
+brokerStart CB_C
+
+--SHELL--
+echo "0.1.Send a subscribe to A"
+echo "==============================================================="
+url="/v1/subscribeContext"
+payload='<?xml version="1.0" encoding="UTF-8"?>
+<subscribeContextRequest>
```

Comment:
I'm afraid XML is a deprecated format since 0.23.0, thus the test should be in JSON.